src/tabs/setup.html

```diff
@@ -128,12 +128,8 @@
                                 <td class="gpsSats"></td>
                             </tr>
                             <tr>
-                                <td i18n="gpsLat"></td>
-                                <td class="gpsLat"></td>
-                            </tr>
-                            <tr class="noboarder">
-                                <td i18n="gpsLon"></td>
-                                <td class="gpsLon"></td>
+                                <td i18n="gpsLatLon"></td>
+                                <td class="latLon"><a href="#" target="_blank">0.0000 deg</a></td>
```

Comment:
@HThuren please fix:

- In Setup tab `latLon` is not defined
- In GPS tab `gpsLat` and `gpsLon` are still being referenced.

```
i18next::translator: missingKey en messages gpsLatLon gpsLatLon
three.module.js:21954 THREE.WebGLRenderer 97
i18next.js:22 i18next::translator: missingKey en messages gpsLat gpsLat
i18next.js:22 i18next::translator: missingKey en messages gpsLon gpsLon
gps.js:314 Online
```

Reply:
Right, this was part of #3486, but not to wait on dialog about that PRm I do another to fix the GPS tab. Same design as on SETUP tab, since the url are the same for both lat and lon.
![image](https://github.com/betaflight/betaflight-configurator/assets/99370924/d7d61621-b67d-4bc7-9f39-7298b3292fe7)